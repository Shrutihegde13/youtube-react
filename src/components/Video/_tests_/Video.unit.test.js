import React from 'react';
import {Video} from '../Video';
import {shallow} from 'enzyme';


test('renders video component correctly',()=> {
   const wrapper = shallow(
       <Video id='HAuXJVI_bUs'/>
    
   );
   expect(wrapper).toMatchSnapshot();

});