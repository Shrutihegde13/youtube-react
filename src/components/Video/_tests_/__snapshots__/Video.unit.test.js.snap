// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders video component correctly 1`] = `
<div
  className="video-container"
>
  <div
    className="video"
  >
    <iframe
      allow="autoplay; encrypted-media"
      allowFullScreen={true}
      className="video-player"
      frameBorder="0"
      src="https://www.youtube.com/embed/HAuXJVI_bUs?autoplay=1"
      title="video"
    />
  </div>
</div>
`;
